homeassistant/components/bluetooth/update_coordinator.py

```diff
@@ -33,14 +35,13 @@ def __init__(
         """Initialize the coordinator."""
         self.hass = hass
         self.logger = logger
-        self.name: str | None = None
         self.address = address
         self.connectable = connectable
         self._cancel_track_unavailable: CALLBACK_TYPE | None = None
         self._cancel_bluetooth_advertisements: CALLBACK_TYPE | None = None
-        self._present = False
```

Comment:
We were initializing this as `False` when the device could actually be available but we haven't received and adv for it since the coordinator was created